megameklab/src/megameklab/printing/reference/ClusterHitsTable.java

```diff
@@ -74,7 +74,9 @@ private void calculateClusterSizes(Entity entity) {
             if (entity instanceof BattleArmor) {
                 for (Mounted mounted : entity.getIndividualWeaponList()) {
                     if (mounted.getType() instanceof MissileWeapon) {
-                        clusterSizes.add(Math.min(40, size * ((MissileWeapon) mounted.getType()).getRackSize()));
+                        for (int i = 1; i <= size; i++) {
```

Comment:
Just being very careful by double checking this is supposed to start at 1? It reads like it is, but better to be safe.

Reply:
It's the number of active troopers. I changed the loop variable name to make it more evident.